validator/project_validator.go

```diff
@@ -222,6 +222,60 @@ func getDistrosForProject(ctx context.Context, projectID string) (ids []string,
 	return ids, aliases, nil
 }
 
+func CheckProject(ctx context.Context, project *model.Project, config *model.ProjectConfig, ref *model.ProjectRef, settings *evergreen.Settings, includeLong, projectRefProvided, projectRefErrored bool) ValidationErrors {
+	isConfigDefined := config != nil
+	verrs := CheckProjectErrors(ctx, project, includeLong)
+	if !projectRefProvided {
+		verrs = append(verrs, ValidationError{
```

Comment:
We can combine this into one warning (also below)

Reply:
I combined them, originally they were different warnings so I kept it to keep behavior- I think it makes sense to combine if you think it isn't a big deal to change the output of it.